docs/topics/mta-web-run-assessment.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-run-assessment_{context}"]
+== Running an assessment on an application
+
+.Prerequisites
+
+* Be logged on to a server {ProductShortName}.
+
```

Comment:
Oops -- {ProductShortName} server. Probably my bad -- might want to run check for server {ProductShortName} or "server MTA" in the preview.